factoryWithThrowingShims.js

```diff
@@ -51,6 +51,7 @@ module.exports = function() {
     objectOf: getShim,
     oneOf: getShim,
     oneOfType: getShim,
+    elementWithType: getShim,
```

Comment:
also let’s name this `elementType`

Reply:
elementType already exist, looks like this PR beated us to it. https://github.com/facebook/prop-types/commit/b67bbd40b5bf8b27475801ecdb7617071ceaccac. I am going to close my PR.

Reply:
elementType already exist, looks like this PR beated us to it. https://github.com/facebook/prop-types/commit/b67bbd40b5bf8b27475801ecdb7617071ceaccac.

Reply:
Whoops, indeed - looks like #211 may have replaced this.